src/rshim_fuse.c

```diff
@@ -721,6 +731,11 @@ static int rshim_fuse_misc_read(struct cuse_dev *cdev, int fflags,
     len -= n;
   }
 
+  n = snprintf(p, len, "%-16s%d (0:no, 1:yes)\n", "FORCE_CMD",
+      bd->force_cmd_pending);
```

Comment:
A question, will the force_cmd_pending be cleared once it succeed, or later taken over again by the other side?

Reply:
It'll be cleared once it's executed, regardless of whether it's successful or not.